### DISCUSSION TITLE
Question about custom cursor position & waveform starting position.

### DISCUSSION THREAD

**Discussion by cdslxc** [katspaugh/wavesurfer.js#3758] [Q&A] [answered]
https://github.com/katspaugh/wavesurfer.js/assets/8092790/999dc3a1-5fc3-4512-9950-121a16639100

Dear maintainer, how can I customize wavesurfer.js like that.
Basically, the cursor is hidden at the end (right-side) of the canvas, and the waveform indicate the current position of the audio.
Is it possible to do it?

**Comment by katspaugh** [comment]
You can hide the cursor by passing `cursorColor: 'transparent'` or `cursorWidth: 0`.

**Reply by cdslxc** [reply]
@katspaugh Certainly, I did that to hide the cursor. Did you notice those first blank bars in my video?
Basically I want to offset the sound wave start outside of the canvas.
  
Initial load or at timestamp 00:00 should looks like:
![CleanShot 2024-06-21 at 18 14 19@2x](https://github.com/katspaugh/wavesurfer.js/assets/8092790/550b5ea0-d559-4e3e-8921-0924d23cbe5a)

as the audio play, the soundwave slide in from right to left:
![CleanShot 2024-06-21 at 18 16 08@2x](https://github.com/katspaugh/wavesurfer.js/assets/8092790/4cb944d7-c177-455f-959a-d9bf34d963b8)

![CleanShot 2024-06-21 at 18 14 30@2x](https://github.com/katspaugh/wavesurfer.js/assets/8092790/68968e34-0b7b-4fa9-a47c-c45feeb4af34)

**Reply by katspaugh** [reply, marked as answer]
Ah, I see. You want to pad an audio track with some silence in the beginning. There's no built-in way to do that with wavesurfer.

**Reply by cdslxc** [reply]
@katspaugh thanks for the instant response. I hope there will be a way to customize the cursor soon.
